nginx_proxy/DOCS.md

```diff
@@ -74,6 +74,10 @@ The filename of the NGINX configuration for the default server, found in the `/s
 
 The filename(s) of the NGINX configuration for the additional servers, found in the `/share` directory.
 
+### Option `customize.root_location` (optional)
```

Comment:
I suggest to name the config `default_root_location`, so it aligns with the server config name.